inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/DynamicWorkloadManagementPage.java

```diff
@@ -353,10 +354,10 @@ protected void onSubmit(AjaxRequestTarget target) {
             PropertyModel.of(aProv,"filter.selected")) {
 
             @Override
-            protected void onUpdate(AjaxRequestTarget ajaxRequestTarget) {
+            protected void onUpdate(AjaxRequestTarget ajaxRequestTarget)
+            {
 
-                if (getDefaultModelObjectAsString().equals("false"))
-                {
+                if (getDefaultModelObjectAsString().equals("false")) {
```

Comment:
Please do not use UI strings for backend logic. Checkbox should have a getModelObject() method which should return boolean.

Reply:
Yes they have, but even though it should return a boolean, it still creates a conversion exception. 
E.g. dateChoices.add(LambdaBehaviour.enabledWhen(() -> unused.getModelObject()) results in an error, with unused.getValue().equals("false") it does not